.github/workflows/ci.yaml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    # Display name for the GHA page
+    name: Linux docker
+
+    runs-on: ubuntu-18.04
+
+    # Is 10 mins enough?
+    timeout-minutes: 10
+
+    steps:
+    - name: Repo checkout
+      uses: actions/checkout@v2
+
+    - name: build
+      run: |
+        cd build/docker/
+        make all
+        make run 
```

Comment:
This might be excessive or not work on GitHub actions? Are there any tests we can run instead?

Reply:
make run is useless, could you remove it
also call make only (all is the default target)